dv/riscv_compliance/README.md

```diff
@@ -62,7 +63,7 @@ How to run RISC-V Compliance on Ibex
    # give the absolute path to the simulation binary compiled in step 1
    export TARGET_SIM=/path/to/your/Vibex_riscv_compliance
 
-   export RISCV_DEVICE=rv32imc
+   export RISCV_DEVICE=I            # Other targets for compliance can be C & M
```

Comment:
Just to confirm: IMC is not a valid option anymore?

Reply:
Yes. Targets in riscv-compliance have changed from rv32imc to I, M, C, Zifencei etc.